### PR TITLE
[IMP] accounting: batch payments sdd

### PR DESCRIPTION
**Pull request by chiaraprattico**
taskid: 3046608

### CODE REVIEW

File: content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst [hunk 1]
```diff
@@ -59,49 +54,42 @@ SEPA Direct Debit can be used as a payment method both on your **eCommerce** or
...
-Direct Debit mandates are closed automatically after their *End Date*. If this field is left blank,
-the mandate keeps being *Active* until it is *Closed* or *Revoked*.
-
-.. image:: batch_sdd/batch-sdd-revoke-close.png
-   :align: center
-   :alt: Close or Revoke an SDD Mandate in Odoo Accounting
+Direct Debit mandates are closed automatically after their **End Date**. If this field is left
+blank, the mandate keeps being **Active** until it is **Closed** or **Revoked**.
 
 Clicking on **Close** updates the mandate’s end day to the current day. This means that invoices
```
Comment (toaa-odoo) [inline content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst:73]:
```suggestion
Clicking on :guilabel:`Close` updates the mandate’s end day to the current day. This means that invoices
```

File: content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst [hunk 2]
```diff
@@ -59,49 +54,42 @@ SEPA Direct Debit can be used as a payment method both on your **eCommerce** or
...
 Close or revoke a mandate 
 -------------------------
 
-Direct Debit mandates are closed automatically after their *End Date*. If this field is left blank,
-the mandate keeps being *Active* until it is *Closed* or *Revoked*.
-
-.. image:: batch_sdd/batch-sdd-revoke-close.png
-   :align: center
-   :alt: Close or Revoke an SDD Mandate in Odoo Accounting
+Direct Debit mandates are closed automatically after their **End Date**. If this field is left
```
Comment (toaa-odoo) [inline content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst:70]:
```suggestion
Direct Debit mandates are closed automatically after their :guilabel:`End Date`. If this field is left
```

File: content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst [hunk 3]
```diff
@@ -114,22 +102,10 @@ process all payments at once.
    required by the SEPA customer-to-bank Implementation Guidelines, which ensures compatibility with
```
Comment (toaa-odoo) [inline content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst:102]:
```suggestion
   required by the SEPA customer-to-bank `implementation guidelines <https://www.europeanpaymentscouncil.eu/document-library/implementation-guidelines/sepa-credit-transfer-customer-psp-implementation>`_, which ensures compatibility with
```

File: content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst [hunk 4]
```diff
@@ -114,22 +102,10 @@ process all payments at once.
...
-*Create Batch Payment*.
-
-.. image:: batch_sdd/batch-sdd-batch.png
-   :align: center
-   :alt: Create a batch payment with all the SDD payments in Odoo Accounting
-
-Odoo then takes you to your *Batch Payment*’s form. Click on *Validate* and download the SDD XML
-file.
+To generate your XML file of multiple pending SDD payments, you can create a batch payment.
+To do so go to :menuselection:`Customers --> Payments`, select the needed payments and then click on
```
Comment (toaa-odoo) [inline content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst:106]:
```suggestion
To do so, go to :menuselection:`Customers --> Payments`, select the needed payments, then click on
```

File: content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst [hunk 5]
```diff
@@ -114,22 +102,10 @@ process all payments at once.
...
+:guilabel:`Action` and finally :guilabel:`Create a batch Payment`. Once you click on
```
Comment (toaa-odoo) [inline content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst:107]:
```suggestion
:guilabel:`Action`, and finally :guilabel:`Create a batch Payment`. Once you click on
```

**Review by toaa-odoo — CHANGES_REQUESTED** [review]
Great PR! Just a small suggestion, you wrote both XML and .XML, it's best to be consistent throughout :)

**Review by toaa-odoo — APPROVED** [review]

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

File: content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst [hunk 6]
```diff
@@ -3,11 +3,11 @@ Batch payments: SEPA Direct Debit (SDD)
...
 SEPA, the Single Euro Payments Area, is a payment-integration initiative of the European Union for
-simplification of bank transfers denominated in EURO. With **SEPA Direct Debit**, your customers can
-sign a **mandate** that authorizes you to collect future payments from their bank accounts. This is
-particularly useful for recurring payments based on a subscription.
+simplification of bank transfers denominated in euros. With **SEPA Direct Debit** (SDD), your
+customers can sign a mandate that authorizes you to collect future payments from their bank
+accounts. This is particularly useful for recurring payments based on a subscription.
 
-You can record customer mandates in Odoo, and generate XML files containing pending payments made
+You can record customer mandates in Odoo, and generate `.xml` files containing pending payments made
```
[the remark this answers is not in the document]
Comment (chiaraprattico) [reply content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst:10]: Not necessarily but it's to be consistent with what's on odoo dbs

[the remark this answers is not in the document]
Comment (AntoineVDV) [reply content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst:10]: Oh god.

File: content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst [hunk 7]
```diff
@@ -33,117 +34,95 @@ SEPA Direct Debit Mandates
 Create a mandate
 ----------------
 
-The SEPA Direct Debit Mandate is the document that your customers sign to authorize you to collect
+The :abbr:`SDD` Mandate is the document that your customers sign to authorize you to collect
```
Comment (AntoineVDV) [inline content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst:37]: Abbreviations must follow [this format](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#abbreviations) to work.

File: content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst [hunk 8]
```diff
@@ -33,117 +34,95 @@ SEPA Direct Debit Mandates
...
-
-.. image:: batch_sdd/batch-sdd-mandate-form.png
-   :align: center
-   :alt: Fill out SEPA Direct Debit in Odoo Accounting
-
-Export the PDF file by clicking on *Print*. It is then up to your customer to sign this document.
-Once done, upload the signed file in the **Original Document** field, and click on *Validate* to
-start running the mandate.
+To create a new mandate, go to :menuselection:`Accounting app --> Customers --> Direct Debit Mandates`,
+click on :guilabel:`Create`, and fill out the form.
```
Comment (AntoineVDV) [inline content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst:41]: Like this:
```suggestion
To create a new mandate, go to :menuselection:`Accounting app --> Customers -->
Direct Debit Mandates`, click on :guilabel:`Create`, and fill out the form.
```

File: content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst [hunk 9]
```diff
@@ -33,117 +34,95 @@ SEPA Direct Debit Mandates
...
+Export the PDF file by clicking on :guilabel:`Print`. It is then up to your customer to sign this
```
Comment (AntoineVDV) [inline content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst:42]: Also, don't force a line break in RST when you don't intend to break the line in HTML.

File: content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst [hunk 10]
```diff
@@ -33,117 +34,95 @@ SEPA Direct Debit Mandates
...
 .. seealso::
    * :doc:`batch`
    * :doc:`../../bank/setup/bank_accounts`
-   * `Odoo Academy: SEPA Direct Debit Mandates (SDD) <https://www.odoo.com/r/Zxs>`_
    * `List of all SEPA countries
-     <https://www.europeanpaymentscouncil.eu/document-library/other/epc-list-sepa-scheme-countries>`_
\ No newline at end of file
+     <https://www.europeanpaymentscouncil.eu/document-library/other/epc-list-sepa-scheme-countries>`_
+   * `Sepa guidelines
+     <https://www.europeanpaymentscouncil.eu/document-library/implementation-guidelines/sepa-credit-transfer-inter-psp-implementation-guidelines>`_
```
Comment (AntoineVDV) [inline content/applications/finance/accounting/receivables/customer_payments/batch_sdd.rst:128]: This should probably be written somewhere, but the guideline is to always have exactly 1 blank line at the end of each file. That's true for many languages (not only RST). This helps editors highlight the code correctly.
```suggestion
     <https://www.europeanpaymentscouncil.eu/document-library/implementation-guidelines/sepa-credit-transfer-inter-psp-implementation-guidelines>`_

```

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

**Comment by AntoineVDV** [comment]
Hello @chiaraprattico, https://github.com/odoo/documentation/pull/2867#discussion_r1016741618 is still unresolved, and unresolved https://github.com/odoo/documentation/pull/2867#discussion_r1016743140 because the change was not applied.

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**